Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -31,32 +30,44 @@ struct SwiftUINativeCheckoutDemo: View {
     }
 
     func checkoutView(_ title: String, _ content: String, _ isFlowComplete: Bool) -> some View {
-        NavigationView {
-            ZStack {
-                VStack(spacing: 16) {
-                    Picker("Checkout ", selection: $checkoutTypeSelection) {
-                        ForEach(CheckoutType.allCases) {type in
-                            Text(type.rawValue)
-                        }
-                    }
-                    Text(title)
-                    Text(content)
-                    Button(isFlowComplete ? "Try Again" : "Start Checkout" ) {
-                        if isFlowComplete {
-                            viewModel.retry()
-                        } else {
-                            startNativeCheckout()
-                        }
+        VStack(spacing: 16) {
+            HStack {
+                Text("Checkout type selection:")
+                Spacer()
+                Picker("", selection: $checkoutTypeSelection) {
+                    ForEach(CheckoutType.allCases) { type in
```

Comment:
Updating the `ForEach` to:
```
ForEach(CheckoutType.allCases, id: \.self) { type in
```
Will get rid of the warning `Non-constant range: not an integer range` for each of these `ForEach` blocks in Xcode.

Reply:
@jaxdesmarais once we update here with checkout with only `orderID`, this `ForEach` will go away

Reply:
@jaxdesmarais the `ForEach` remains but now its for `ShippingType`. Im not getting the warning you mentioned, where should i see it?

Reply:
Not sure which Xcode version you are using but I was seeing it in Xcode 14 when running through the demo.

Reply:
I'll see if i need to update my xcode, but i pushed the change anyway!